deploy/helm/sumologic/values.yaml

```diff
@@ -1694,13 +1694,41 @@ kube-prometheus-stack:
     ## Use the GCR repo, it's more recent and has ARM images starting from 1.9.8
     image:
       repository: k8s.gcr.io/kube-state-metrics/kube-state-metrics
-      tag: v1.9.8
-    # Do not collect metrics for CertificateSigningRequest or Ingress resources.
-    # This is because the v1 version of kube-state-metrics uses API versions
-    # that are removed in Kubernetes v1.22.
+      tag: v2.3.0
+    ## Do not collect metrics for CertificateSigningRequest or Ingress resources.
+    ## This is because the v1 version of kube-state-metrics uses API versions
+    ## that are removed in Kubernetes v1.22.
+    ##
+    ## See the following configuration for full list of collectors:
+    ## https://github.com/prometheus-community/helm-charts/blob/kube-state-metrics-4.13.0/charts/kube-state-metrics/values.yaml#L174-L204
     collectors:
-      certificatesigningrequests: false
-      ingresses: false
+      # - certificatesigningrequests
```

Comment:
The comment above explains that we have been removing the two collectors due to kube-state-metrics `v1` using old API versions. I assume `v2` of kube-state-metrics uses current API versions, so perhaps we don't need to remove these resources?

Reply:
Makes sense. Didn't thought about that

Reply:
Do you think we should remove it in migrations scripts, or leave it because user wanted this changed as they copied it :upside_down_face: ?

Reply:
I think we should remove it 🙂

Reply:
What if someone disable something more?

Reply:
Here's my thinking about migration:

1. If the user does not have the property in their `values.yaml` file, there's nothing to migrate.
2. If the user has the property in their `values.yaml` file, we should migrate the property to keep the user's modifications.

There's of course the unfortunate case of the user copying all the default properties into their `values.yaml` file (using the default `values.yaml` file as their custom config) - I'm not sure how we should proceed here. It's tempting to "clean up" the `values.yaml` file before migrating by removing all the properties from it that are unchanged. I'm not sure this is the right thing to do though.

Reply:
perfecto, nothing to change in migration tool :joy: